src/lambdas/initiateDataRequest/zendeskTicketDiffersFromRequest.ts

```diff
@@ -69,6 +69,40 @@ const getZendeskCustomSpaceSeparatedStringAsArray = (
   return mapSpaceSeparatedStringToList(fieldValue)
 }
 
+const datesMatch = (
+  ticketDetails: ZendeskTicket,
+  requestParams: DataRequestParams
+) => {
+  const ticketLegacyDateField = getZendeskCustomFieldValue(
+    ticketDetails,
+    getEnvAsNumber('ZENDESK_FIELD_ID_DATE_FROM')
+  )
+  const ticketDates = getZendeskCustomSpaceSeparatedStringAsArray(
+    ticketDetails,
+    getEnvAsNumber('ZENDESK_FIELD_ID_DATES')
+  )
+  if (ticketLegacyDateField) {
+    const returnValue = matchArrayParams(
+      [ticketLegacyDateField as string],
+      requestParams.dates
+    )
+    if (!returnValue) {
+      console.log(
```

Comment:
This will get logged at the end of `ticketAndRequestDetailsDiffer` - if you think this is still useful to log could you use `logger` instead of `console.log`

Reply:
You've cleverly spotted my debug code that wasn't supposed to be there... Removed now